openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -1480,6 +1481,7 @@ static int SSL_cert_verify(X509_STORE_CTX *ctx, void *arg) {
     ret = result == X509_V_OK ? 1 : 0;
     return ret;
 }
+#endif // OPENSSL_IS_BORINGSSL
 
 #ifdef OPENSSL_IS_BORINGSSL
```

Comment:
@creamsoup maybe it would be even "cleaner" to compile the above with this one using an `#else` . WDYT ?

Reply:
yeah didn't realized that. thanks.